boxsdk/object/item.py

```diff
@@ -725,3 +744,20 @@ def remove_classification(self):
             `bool`
         """
         return self.metadata('enterprise', self._classification_template_key).delete()
+
+    @staticmethod
+    def validate_item_id(item_id):
+        """
+        Validates an item ID is a digit
+
+        :param item_id:
+        :type classification:
+            `str`
+        :raises:
+            BoxException: if item_id is not a digit
+        :returns:
+        :rtype:
+            None
+        """
+        if not item_id.isdigit():
+            raise BoxException("Invalid item ID")
```

Comment:
Anything more precise we can use like ValueError?

Reply:
Good point. What do you think about adding a BoxValueError to boxsdk.exception that subclasses ValueError and raising that?